src/css/mixins.scss

```diff
@@ -0,0 +1,120 @@
+// Helper mixins TODO: get rid of these mixins: https://github.com/noraesae/perfect-scrollbar/issues/431
```

Comment:
Please remove TODO comment here. I would consider removing these mixins, but the comment itself is redundant as there is already an issue reported. No need for further alarming.